processor/src/main/kotlin/permissions/dispatcher/processor/PermissionsProcessor.kt

```diff
@@ -52,9 +52,7 @@ class PermissionsProcessor : AbstractProcessor() {
     }
 
     override fun process(annotations: Set<TypeElement>, roundEnv: RoundEnvironment): Boolean {
-        // Create a RequestCodeProvider which guarantees unique request codes for each permission request
-        val requestCodeProvider = RequestCodeProvider()
-
+        val requestCodeProvider = RequestCodeProvider
```

Comment:
Do we still need this val? If the `RequestCodeProvider` is accessible anyway, we don't have to pass it around anymore, I think

Reply:
I just kept current interface deliberately since there's no guarantee to solve the issue. After confirming I'd like to fix more fundamentally.

Reply:
OK!